lib/integrations/pivotal_tracker.rb

```diff
@@ -1,7 +1,43 @@
 module Integrations
   class PivotalTracker < Base
     # NOTE: Pivotal Tracker integration development still underway
-    include Integrations::MessageFormatter
+
+    def message_text
+      message = self.send(event_type.dup.concat("_message").to_sym)
+      "Your Rainforest Run ##{run[:id]}#{run_description} - #{payload[:frontend_url]}) #{message}"
+    end
+
+    def run_description
+      run[:description].present? ? ": #{run[:description]}" : ""
+    end
+
+    def run_completion_message
+      "is complete!"
+    end
+
+    def run_error_message
+      "has encountered an error!"
+    end
+
+    def webhook_timeout_message
+      "has timed out due to a webhook failure!\nIf you need a hand debugging it, please let us know via email at help@rainforestqa.com."
+    end
+
+    def run_test_failure_message
+      "has a failed a test!"
+    end
+
+    def humanize_secs(seconds)
```

Comment:
this is actually a great utility method. might be a good idea to have a way for integrations to share it.